docker/mongodb/scripts/run.sh

```diff
@@ -1,22 +1,35 @@
 #!/bin/bash
 
+
+run_mongo_commandline () {
+if (( $(echo "$MONGO_MAJOR > 5.9" | bc -l) )); then
+    nohup gosu mongodb mongosh admin --eval "help" > /dev/null 2>&1
+else
+    nohup gosu mongodb mongo admin --eval "help" > /dev/null 2>&1
+fi
+}
+
 sleep 5
 
 chown -R mongodb:mongodb /home/mongodb
 
 nohup gosu mongodb mongod --dbpath=/data/db &
 
-nohup gosu mongodb mongo admin --eval "help" > /dev/null 2>&1
+run_mongo_commandline
+
 RET=$?
 
 while [[ "$RET" -ne 0 ]]; do
   echo "Waiting for MongoDB to start..."
-  mongo admin --eval "help" > /dev/null 2>&1
+  run_mongo_commandline
   RET=$?
   sleep 2
 done
 
-bash /home/mongodb/scripts/setup_user.sh
-
+if (( $(echo "${MONGO_MAJOR} > 5.9" | bc -l) )); then
+    bash /home/mongodb/scripts/setup_user_newer_ver.sh
+else
+    bash /home/mongodb/scripts/setup_user.sh
```

Comment:
So the only difference is mongo cli? Can we just define it as a var once and avoid duplicating the scripts?

```
MONGO=$((echo "${MONGO_MAJOR} > 5.9" | bc -l) && echo "mongo" || echo "mongosh")
```

Then use ${MONGODB} in other places

Reply:
working on it.

Reply:
I tried the expression above. For the truth case, meaning version 6.0 and above it returns "1 mongosh" hence, I am simplifying the expression and making it a bit more verbose.